clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -1641,13 +1667,18 @@ scheduler_impl_tmpl_t<RecordType, ReaderType>::read_kernel_sequences(
                     "Invalid " + sequence_type + " sequence found with default key";
                 return sched_type_t::STATUS_ERROR_INVALID_PARAMETER;
             }
-            if (!sequence[sequence_key].empty()) {
+            if (!sequence[sequence_key].records.empty()) {
                 error_string_ += "Duplicate " + sequence_type + " sequence found";
                 return sched_type_t::STATUS_ERROR_INVALID_PARAMETER;
             }
         }
-        if (in_sequence)
-            sequence[sequence_key].push_back(record);
+        if (in_sequence) {
+            sequence[sequence_key].records.push_back(record);
+            if (sequence[sequence_key].first_pc == 0 &&
```

Comment:
Is 0 a valid sentinel? I guess we'd never have an ifetch from 0 on any modern system. A program that jumps to 0 we should support: but it would fault and either die or run a handler. We'd have the culprit indirect branch as the prior PC and the handler as the next PC? What would we store as the target of the indirect branch?? We couldn't figure it out in raw2trace?